tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -652,3 +654,71 @@ def test_flow_as_unauthenticated_existing_user_legacy_identity_migration(self):
         # Ensure the ident was migrated from the legacy identity
         updated_ident = AuthIdentity.objects.get(id=user_ident.id)
         assert updated_ident.ident == "foo@new-domain.com"
+
+    @override_settings(SENTRY_SINGLE_ORGANIZATION=True)
+    @with_feature({"organizations:create": False})
+    def test_basic_auth_flow_as_invited_user(self):
+
+        user = self.create_user("foor@example.com")
+        self.create_member(organization=self.organization, user=user)
+        member = OrganizationMember.objects.get(organization=self.organization, user=user)
+        member.email = "foor@example.com"
+        member.user = None
+        member.save()
+
+        # session = self.client.session
```

Comment:
i can't get the session value setting to work here but i'd like to test when `_next` is an OrgView.